package.json

```diff
@@ -23,9 +20,14 @@
     "test": "vitest run",
     "test:coverage": "vitest run --coverage && open ./coverage/index.html",
     "test:watch": "vitest",
+    "prep:e2e": "npm run clean && npm run build && start-server-and-test start:mocks http://localhost:3000",
+    "test:e2e": "npm run prep:e2e \"playwright test\"",
+    "test:e2e:chrome": "npm run prep:e2e \"playwright test --project=chromium\"",
+    "test:e2e:firefox": "npm run prep:e2e \"playwright test --project=firefox\"",
+    "test:e2e:webkit": "npm run prep:e2e \"playwright test --project=webkit\"",
     "prepare": "husky install",
-    "typecheck": "tsc -b",
-    "validate": "run-p test lint typecheck test:e2e:run",
+    "typecheck": "tsc -b && tsc -b e2e",
+    "validate": "run-p test lint typecheck test:e2e",
```

Comment:
We lost checking that the e2e tests are typed correctly, and the validate script was invalid as written

Reply:
Thanks for adding that back.